### DISCUSSION TITLE
How to help

### DISCUSSION THREAD

**Discussion by YSCohen** [hylang/hy#2266] [General]
Hey!
I'm new to Hy, and semi-new to Python. Where is a good start for beginner contributions, or what should I do to familiarize myself with the codebase?

**Comment by Kodiologist** [comment]
Would you like to take a stab at #1383? "Shadowing" a core macro means adding a function for it to `pyops.hy` so that it can be used by a higher-order function. This sort of task would get you working with the code and writing tests but shouldn't be too hard.

**Comment by YSCohen** [comment]
@Kodiologist so I would be implementing `cut` in `pyops.hy`?

**Reply by Kodiologist** [reply]
Right.
